### DISCUSSION TITLE
Teensy 4.0 compatibility? (no Teensy 3.2 availability)

### DISCUSSION THREAD

**Discussion by cbradburne** [luni64/TeensyStep#147] [General]
Hi,

As Teensy 3.2 boards have not been available for some time (UK), I was wondering what’s the current state of teensystep for Teensy 4.0?
I don’t need any extra features or speed, just an alternative to 3.2 that I can actually buy. 

Thanks,
Colin

**Comment by tobiaskebis-zebra** [comment]
similar here. I would also like to use Teensy 4.0
